docs/Set-PSReadLineOption.md

```diff
@@ -612,6 +615,28 @@ Accept pipeline input: false
 Accept wildcard characters: false
 ```
 
+### -PredictionStyle
+
+Specifies how PSReadLine should render the predictive suggestions.
+
+Valid values are:
+
+-- None: disable the predictive suggestion feature
+
+-- Concise: render the predictive suggestion in a concise view
+
```

Comment:
Without knowing what other styles will be added, it's not clear that `Concise` is sufficiently descriptive. I'd probably be happier if it was called `Default` (preferred) or maybe `Simple`.

Reply:
I will change it to `Default`